README.md

```diff
@@ -217,6 +217,8 @@ Options, and their defaults
   bodyParser: {
     limit: 1024 * 1000,
   },
+  // number of cpus hypernova will use. 0 to use all available
+  cpus: 0,
```

Comment:
I think it is very confusing to have zero mean "all" - I'd expect Infinity to mean "all".